docs/runtime-agent/03-02-configuring-runtime.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Configuring the Runtime
+type: Details
+---
+
+> **NOTE:** To represent API and Event Definitions of the connected Applications on Runtime, Open Service Broker API usage is recommended.
```

Comment:
```suggestion
> **NOTE:** To represent API and Event definitions of the Applications connected to a Runtime, Open Service Broker API usage is recommended.
```

Reply:
I am not sure about this one, though 🤔

Reply:
I've consulted a dev about it. Apart from `Event Definitions`, the suggestion is fine 👍